applications/projects/creating-project-other-user.adoc

```diff
@@ -2,10 +2,11 @@
 = Creating a project as another user
 include::modules/common-attributes.adoc[]
 :context: creating-project-other-user
+
 toc::[]
 
 Impersonation allows you to create a project as a different user.
 
 include::modules/authentication-api-impersonation.adoc[leveloffset=+1]
 
-include::modules/impersonation-project-creation.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/impersonation-project-creation.adoc[leveloffset=+1]
```

Comment:
Where you see the last line in a file being removed and added in this PR, this is the script that I ran adding new lines to the end of a file where one did not previously exist. It was a side effect of the script, but it is useful.